neume.mjs

```diff
@@ -22,6 +22,10 @@ const argv = yargs(hideBin(process.argv))
   try {
     await boot(crawlPath, config);
   } catch (err) {
-    console.error(err.toString());
+    console.error(
+      "An unhandled error has bubbled up which shouldn't have happened. Force closing the application."
+    );
+    console.error(err);
+    process.exit();
```

Comment:
```suggestion
    process.exit(1);
```

Reply:
https://nodejs.org/api/process.html#processexitcode we want to use failure code 1